.github/workflows/main.yml

```
@@ -0,0 +1,26 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 9, 10, 11]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Use java ${{ matrix.java }}
+        uses: actions/setup-java@v1
```

It's harder to find documentation for `actions/setup-java@v1` now, since v2 has been promoted to main. Can you include a link here?

```suggestion
        # See documentation here: https://github.com/actions/setup-java/blob/81290cdb8c027e945e66c40b9efd912511c40859/README.md
        uses: actions/setup-java@v1
```